package/icu/icu.hash

```diff
@@ -3,3 +3,4 @@ md5	976734806026a4ef8bdd17937c8898b9	icu4c-57_1-src.tgz
 md5	fac212b32b7ec7ab007a12dff1f3aea1	icu4c-58_2-src.tgz
 # Calculated based on the hash above
 sha256	ff8c67cb65949b1e7808f2359f2b80f722697048e90e7cfc382ec1fe229e9581	icu4c-57_1-src.tgz
+sha256  2b0a4410153a9b20de0e20c7d8b66049a72aef244b53683d0d7521371683da0c icu4c-58_2-src.tgz
```

Comment:
I think this is a duplicate of https://github.com/WebPlatformForEmbedded/buildroot/pull/79 ? I just merged that one in.

Reply:
Will rebase against latest master